src/sidebar/store/test/util-test.js

```diff
@@ -109,19 +133,19 @@ describe('reducer utils', function() {
 
   describe('#bindSelectors', function() {
     it('bound functions call original functions with current value of getState()', function() {
-      const annotations = [{ id: 1 }];
-      const getState = sinon.stub().returns({ annotations: annotations });
-      const bound = util.bindSelectors(
-        {
-          countAnnotations: fixtures.countAnnotations,
+      const getState = sinon.stub().returns({
+        namespace1: {
+          annotations: [{ id: 1 }],
         },
-        getState
-      );
-
-      assert.equal(bound.countAnnotations(), 1);
-
-      getState.returns({ annotations: annotations.concat([{ id: 2 }]) });
-      assert.equal(bound.countAnnotations(), 2);
+        namespace2: {
+          annotations: [{ id: 1 }],
+        },
+      });
+      const bound = util.bindSelectors(fixtures.selectors, getState);
+      // test namespaced selector
+      assert.equal(bound.countAnnotations1(), 1);
```

Comment:
`countAnnotations1` is a non-namespaced selector rather than a namespaced one. Conversely `countAnnotations2` is a namespaced selector.

Reply:
good catch!